### PR TITLE
Add Lambda reader function which packages events for SQS out to aggre…

### PR DESCRIPTION
**Pull request by janetlee**
…gation service

If anyone is out there during the holidays working, I could use some 👀 on this PR. All it does is take an event record and package all the events for the view instance id and push them out the door. It may need some tweaks/refactor when I get a better sense of how the SNS topic/queue functions for filtering END events.

### CODE REVIEW

File: server/reader/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+var cassandra = require('./node_modules/cassandra-driver');
+var client = new cassandra.Client({ contactPoints: ['172.31.14.84'], keyspace: 'events' });
+var QUEUE_URL = 'https://sqs.us-east-2.amazonaws.com/331983685977/packaged-events';
+var AWS = require('aws-sdk');
+var sqs = new AWS.SQS({region : 'us-east-2'});
+
+module.exports.handler = function(eventRecord) {
+  var query = 'SELECT * FROM Events.event WHERE videoid= ? AND viewinstanceid = ? allow filtering;';
+  var queryParams = [eventRecord.videoId, eventRecord.viewInstanceId];
```
Comment (kaiserawu) [inline server/reader/index.js:9]: Is it necessary to search based on video id and view instance id? The view instance id should be unique, so it doesn't seem necessary.

**Review by kaiserawu — COMMENTED** [review]
See posted comment before approving, otherwise it all seems dandy.

File: server/reader/index.js (hunk 1 above)
Comment (janetlee) [reply server/reader/index.js:9]: I was not clear on if the view instance ID is unique. I cannot remember which service is issuing it so I didn't want to assume it was always unique.

Comment (janetlee) [reply server/reader/index.js:9]: I clarified this with Dono. He's not finalized the ID, but it probably will be unique. I will update with a new commit soon.

Comment (janetlee) [reply server/reader/index.js:9]: Closing this to repull....

**Pull request closed without merging**